lib/std/os/linux.zig

```diff
@@ -350,6 +366,28 @@ pub const O = switch (native_arch) {
         TMPFILE: bool = false,
         _: u6 = 0,
     },
+    .loongarch64 => packed struct(u32) {
```

Comment:
Same here; identical to the `x86`/`riscv32`/`riscv64` prong.

Reply:
Same here

- https://elixir.bootlin.com/linux/v5.19/source/include/uapi/asm-generic/fcntl.h